src/js/core.js

```diff
@@ -73,10 +73,12 @@
             } else if (MediumEditor.util.isKey(event, MediumEditor.util.keyCode.ENTER)) {
                 // hitting return in the begining of a header will create empty header elements before the current one
                 // instead, make "<p><br></p>" element, which are what happens if you hit return in an empty paragraph
-                p = this.options.ownerDocument.createElement('p');
-                p.innerHTML = '<br>';
-                node.previousElementSibling.parentNode.insertBefore(p, node);
-                event.preventDefault();
+                if (!this.options.disableDoubleReturn) {
```

Comment:
Should this `!this.options.disableDoubleReturn` check be merged into the existiing outer-if?

ie:
```js
} else if (!this.options.disableDoubleReturn &&
              MediumEditor.util.isKey(event, MediumEditor.util.keyCode.ENTER)) {
    p = this.options.ownerDocument.createElement('p');
    // rest of code
}
              
```